<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BehaviorView</class>
 <widget class="QWidget" name="BehaviorView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabInitiate">
      <attribute name="title">
       <string>initiate</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QWidget" name="widget" native="true">
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>96</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>time interval(s)</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>9</y>
            <width>96</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Max num of nodes</string>
          </property>
         </widget>
         <zorder>pushButton</zorder>
         <zorder>label_3</zorder>
         <zorder>label</zorder>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widget_2" native="true">
         <widget class="QLineEdit" name="lineEdit_maxNode">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_timeInterval">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>61</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QWidget" name="widget_3" native="true">
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>9</x>
            <y>9</y>
            <width>96</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>time duration(s)</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>144</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Max concentration(mol/L)</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QWidget" name="widget_4" native="true">
         <widget class="QLineEdit" name="lineEdit_timeDuration">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>10</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit_maxConcentration">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>71</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>1</string>
          </property>
         </widget>
         <zorder>lineEdit_timeDuration</zorder>
         <zorder>lineEdit_maxConcentration</zorder>
         <zorder>pushButton</zorder>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>next</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDefine">
      <attribute name="title">
       <string>defining</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget_define"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>next</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBehavior">
      <attribute name="title">
       <string>ideal behavior</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="font">
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabValue">
          <attribute name="title">
           <string>ideal value</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget_behavior"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>view graphic</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tabGraphic">
          <attribute name="title">
           <string>coarse graphic</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QGraphicsView" name="graphicsView"/>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="pushButton_4">
             <property name="text">
              <string>view value</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
